R/pctr_score_companies.R

```diff
@@ -41,7 +41,7 @@ pctr_score_companies <- function(scored_activities, companies) {
 
   # join by activity_product_uuid and other joint columns from companies with scored_activities
   companies_scores <- companies |>
-    inner_join(scored_activities, by = c("activity_product_uuid", "ei_activity", "unit"))
+    left_join(scored_activities, by = c("activity_product_uuid", "ei_activity", "unit"))
```

Comment:
Here is what I assume is a bug.

Reply:
It would be best for the author @Tilmon to confirm, but it seems like a reasonable assumtion.

From `?dplyr::inner_join()`:

> An inner_join() only keeps observations from x that have a matching key in y.
> The most important property of an inner join is that unmatched rows in either input are not included in the result. This means that generally inner joins are not appropriate in most analyses, because it is too easy to lose observations.